exaudfclient/base/python/python3/python_ext_dataframe.cc

```diff
@@ -1167,6 +1268,11 @@ void emit(PyObject *resultHandler, std::vector<ColumnInfo>& colInfo, PyObject *d
                         handleEmitPyInt(c, r, columnArrays, pyColSetMethods, colInfo, colTypes, resultHandler, pyValue, pyResult, pySetNullMethodName);
                         break;
                     }
+                    case PY_FLOAT:
```

Comment:
Does it mean we drop support for NPY_FLOAT16, and support only NPY_FLOAT32/NPY_FLOAT16 and PY_FLOAT?